Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="denys.kotovskiy@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
```

Comment:
Consider using `COPY requirements.txt /app/` to ensure that the `requirements.txt` file is copied to the correct directory specified by `WORKDIR`. This helps maintain consistency and clarity in the file structure.